### PR TITLE
Change debug information format

### LINKED ISSUE #22632: Fix warning "LNK4075: ignoring '/EDITANDCONTINUE' due to '/SAFESEH' specification"
Occurs during native build phase of Windows builds. Full text:
``` text
gtestd.lib(gtest-all.obj) : warning LNK4075: ignoring '/EDITANDCONTINUE' due to '/SAFESEH' specification [F:\workspace\_work\1\s\src\Servers\IIS\AspNetCoreModuleV2\CommonLibTests\CommonLibTests.vcxproj]
```

I believe #22017 introduced this warning though I'm not sure why. Warning is present in aspnetcore-ci build [20200529.33](https://dev.azure.com/dnceng/public/_build/results?buildId=664695) of master just after #22017 went in as well as [20200529.29](https://dev.azure.com/dnceng/public/_build/results?buildId=664584), a validation build for that PR but not in the builds of master before then, at least where I checked.

Part of #22240.

### PR DESCRIPTION
**Pull request by jkotalik**
Fixes https://github.com/dotnet/aspnetcore/issues/22632

Still trying some things, but I believe changing the debug information format for gtest should fix the issue.

### CODE REVIEW

**Review by javiercn — APPROVED** [review]
I have no context, but I trust you know what you are doing!

File: src/Servers/IIS/AspNetCoreModuleV2/gtest/gtest.vcxproj [hunk 1]
```diff
@@ -179,4 +181,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
+</Project>
```
Comment (pranavkm) [inline src/Servers/IIS/AspNetCoreModuleV2/gtest/gtest.vcxproj:184]: Why did you eat the newline?

Comment (jkotalik) [reply src/Servers/IIS/AspNetCoreModuleV2/gtest/gtest.vcxproj:184]: It just automatically did when I saved the file in VS.

**Comment by jkotalik** [comment]
This worked: https://dev.azure.com/dnceng/public/_build/results?buildId=679121&view=logs&jobId=efda9e5f-cf98-536e-4181-9b6c13ac35b3&j=efda9e5f-cf98-536e-4181-9b6c13ac35b3&t=de5280bb-c511-517b-1959-798dcf9a84da. 

0 warnings for CodeCheck.

**Pull request merged**